lib/os.js

```diff
@@ -179,24 +179,18 @@ platform[SymbolToPrimitive] = () => process.platform;
  * @returns {string}
  */
 function tmpdir() {
-  let path;
   if (isWindows) {
-    path = process.env.TEMP ||
+    let path = process.env.TEMP ||
            process.env.TMP ||
            (process.env.SystemRoot || process.env.windir) + '\\temp';
     if (path.length > 1 && StringPrototypeEndsWith(path, '\\') &&
         !StringPrototypeEndsWith(path, ':\\'))
       path = StringPrototypeSlice(path, 0, -1);
```

Comment:
Can we use const with early return?

```suggestion
    const path = process.env.TEMP ||
           process.env.TMP ||
           (process.env.SystemRoot || process.env.windir) + '\\temp';
    if (path.length > 1 && StringPrototypeEndsWith(path, '\\') &&
        !StringPrototypeEndsWith(path, ':\\'))
      return StringPrototypeSlice(path, 0, -1);
```

Reply:
Yeah, I prefer to do it in a follow up pull-request. It finally have a 100% passing CI run :-)